components/kserve/serverless_setup.go

```diff
@@ -32,8 +33,9 @@ func (k *Kserve) configureServerlessFeatures() feature.FeaturesProvider {
 
 		servingNetIstioSecretFilteringErr := feature.CreateFeature("serverless-net-istio-secret-filtering").
 			For(handler).
+			ManifestSource(Templates.Files).
 			Manifests(
-				path.Join(feature.ServerlessDir, "serving-net-istio-secret-filtering.patch.tmpl"),
+				path.Join(baseDir, "serving-net-istio-secret-filtering.patch.tmpl.yaml"),
```

Comment:
Worth making it part of `Templates` struct? I also wonder if this is a good name. After all it's not only templates we are keeping there.

Reply:
renamed the struct and included `Resources.BaseDir` on top of the const `baseDir` to be used (so the const can still be used in creating the fields for the subpaths